android/app/src/main/java/com/reactnativenavigation/views/sharedElementTransition/SharedElementTransition.java

```diff
@@ -80,7 +80,7 @@ private void saveTextViewSpannedText(final TextView view) {
             @Override
             public void run() {
                 if (spannableText == null) {
-                    spannedText = (SpannedString) view.getText();
+                    spannedText = new SpannedString(view.getText());
```

Comment:
Why create a new SpannedString? Is there a performance hit from creating a new instance of SpannedString?